paasta_tools/kubernetes_tools.py

```diff
@@ -4354,6 +4452,19 @@ def get_kubernetes_secret_env_variables(
     return decrypted_secrets
 
 
+def create_temp_exec_token(kube_client: KubeClient, namespace: str, user: str):
```

Comment:
The `user` parameter is not used here. Is the plan to include it in the service account name if we go for creating ephemeral service accounts?